.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,577 @@
+		"neutral-5": #69778e,
+		"neutral-6": #4f5c71,
+		"neutral-7": #384355,
+		"neutral-8": #232b39,
+		"neutral-9": #10151c,
+		"neutral-10": #000000,
+		"spacing-1": 0.25rem,
+		// 4px
+		"spacing-2": 0.5rem,
+		// 8px
+		"spacing-3": 1rem,
+		// 16px
+		"spacing-4": 1.5rem,
+		// 24px
+		"spacing-5": 2rem,
+		// 32px
+		"spacing-6": 2.5rem,
+		// 40px
+		"spacing-7": 3rem,
+		// 48px
+		"spacing-8": 3.5rem,
+		// 56px
+		"spacing-9": 4rem,
+		// 64px
+		"spacing-10": 4.5rem,
+		// 72px
+		"spacing-11": 5rem,
+		// 80px
+		"spacing-12": 5.5rem,
+		// 88px
+		"spacing-13": 6rem,
+		// 96px
+		"spacing-14": 6.5rem,
+		// 104px
+		"spacing-15": 7rem,
+		// 112px
+		"spacing-16": 7.5rem,
+		// 120px
+		"font-weight-1": 100,
+		"font-weight-2": 200,
+		"font-weight-3": 300,
+		"font-weight-4": 400,
+		"font-weight-5": 500,
+		"font-weight-6": 600,
+		"font-weight-7": 700,
+		"font-weight-8": 800,
+		"font-weight-9": 900,
+		"font-size-1": 0.625rem,
+		// 10px
+		"font-size-2": 0.75rem,
+		// 12px
+		"font-size-3": 0.875rem,
+		// 14px
+		"font-size-4": 1rem,
+		// 16px
+		"font-size-5": 1.125rem,
+		// 18px
+		"font-size-6": 1.1875rem,
+		// 19px
+		"font-size-7": 1.25rem,
+		// 20px
+		"font-size-8": 1.375rem,
+		// 22px
+		"font-size-9": 1.5rem,
+		// 24px
+		"font-size-10": 1.75rem,
+		// 28px
+		"font-size-11": 2rem,
+		// 32px
+		"font-size-12": 2.25rem,
+		// 36px
+		"font-size-13": 2.56rem,
+		// 41px
+		"font-size-14": 3rem,
+		// 48px
+		"font-size-15": 3.25rem,
+		// 52px
+		"line-height-1": 0.875rem,
+		// 14px
+		"line-height-2": 1rem,
+		// 16px
+		"line-height-3": 1.25rem,
+		// 20px
+		"line-height-4": 1.5rem,
+		// 24px
+		"line-height-5": 2rem,
+		// 32px
+		"line-height-6": 2.125rem,
+		// 34px
+		"line-height-7": 2.5rem,
+		// 40px
+		"line-height-8": 3rem,
+		// 48px
+		"line-height-9": 3.5rem,
+		// 56px
+		"line-height-10": 3.875rem,
+		// 62px
+		"box-shadow-1": 0px 0px 16px rgba(25, 25, 25, 0.4),
+		"z-index-1": 1,
+		"z-index-2": 10,
+		"z-index-3": 50,
+		"z-index-4": 100,
+		"z-index-5": 500,
+		"border-radius-1": 0.25em,
+	),
+	$alias: (
+		"layout-max-width": 1440,
+		"layout-content-max-width": 1216,
+		//
+		// --layout-content-scale-width identifies the inner document width for header and content
+		// alignment.  It accounts for a minimum width that has (--global-spacing-5 / 2) margins
+		//
+		"layout-content-scale-width":
+			min(
+				calc(100vw - var(--global-spacing-5)),
+				calc(var(--layout-content-max-width) / var(--layout-max-width) * 100%)
+			),
+		"primary-color": var(--global-neutral-10),
+		"primary-font-family": "Inter",
+		"secondary-font-family": "Lora",
+	),
 	$tokens: (
 		default: (
-			components: (
-				stack: (
-					gap: 1rem,
-				),
-				stack-divider-vertical: (
-					border: none,
-					border-bottom-width: 2px,
-					border-bottom-style: solid,
-					border-bottom-color: #712d2d,
-					height: 1px,
+			// DESIGN ASSUMPTION: Attribution does not require component styling.
```

Comment:
interesting looks like the comment is causing the new line due to a known prettier bug. no worries -- once the comments/questions are done, they should go away  

playground
https://prettier.io/playground/#N4Igxg9gdgLgprEAuEABArgZzgAgDogB0hA9MSZmJpoZdQTgO4CWMAFjgBR5Q584kSOAGY8+eGABIAhgBtm0zEi48JakLOkBPCOhgBaALbSAHvpYATdgWUBGACz2ADABpVMdZp179kWAgNjM0trZBxbACZbADY3XnwPDw1tXUDTc2YrNhtwx1d3TxSfP3hYI3SQ7LDImLjxRIIvVPLgzNC7PLi1JKbi6FK01qycmtixHABKF3x4mSgIdjgAJ305BUwM9n15yENDAOVueO7GosGM4erOgp6z336Alov28Kix44bk73PKkeuoE5fZolR5BZ5VOxvOoJQrfJ6-K7OLqfXoGEFlMEIyG1cYTXEgFwgCAABxgzGgmGQoGkSyWEEYAAUaQhKShpAA3CCZAkgABGS2kYAA1nAYABlaT7AAyzCgcGQwjk2EJ-MFIvFxMFsoA5sgYEt0HBCXBDLy4BYLOapdIoNr0NJtXAAGIQJbGGBk23IEDSPQQHlsGCGWQAdTYrDgmE1YDgYpZrGY7NYWm9VEphNl2CWMAZAu1xgVSqNIAAVpgTGKdbI4ABFdALeVIRWyZUgTVLLPeujpttLWUwENtNjIAAcrl7EGwIYFxO9xKWkeW7PlhIAjvX4LmSayfRs5ebzTyF+vmAvcw6C02i4TsIZmHqDcXMFXaxvG83WzBpLzB1lkBFCX1aRmHkW0AGEID2aRvXmOUeSwOAABVv1ZD9i3ZQ0AEkoEtWAxTAPtSQAQRwsUYC0atCxbYsyxMAAhAVhVFCVpVld8iwAXw4oA

https://github.com/prettier/prettier/issues/4659

Reply:
if we remove the comment here, then the formatting will be as before `:(`